docs/delegation_design_spec/delegation_design_spec.tex

```diff
@@ -491,18 +488,19 @@ \subsection{Overview of Delegation}
 diagram of the system level entities involved in the implementation of
 delegation:
 \begin{itemize}
-\item Each address contains a single payment key. However the same payment key
+\item Each normal pubkey address\footnote{As opposed to script addresses}
+  contains a single payment key. However the same payment key
   can be used by multiple addresses (where addresses differ among each other in
   the delegation information).
 \item Many addresses can refer to the same stake key. For instance, different
   outputs of a transaction can specify the that the stake associated with those
   output should be transferred to the same stake key.
-\item Each stake key have a unique reward account associated with it, and a
-  reward account can be associated to only one stake key.
+\item Each stake key has a unique reward account associated with it, and a
+  reward account is only associated with the one stake key.
```

Comment:
just "with" ?

Reply:
I think using `with the` adds emphasis that it's a one-to-one correspondence.